tests/m2m_signals/tests.py

```diff
@@ -377,6 +420,11 @@ def test_m2m_relations_with_self(self):
             self.m2m_changed_signal_receiver, Person.friends.through
         )
 
+    def test_m2m_relations_with_self_add_friends(self):
+        expected_messages = []
```

Comment:
I think you can do without using a temporary variable to store the messages.

Just use the value directly in the assert.